integration-cli/docker_cli_exec_test.go

```diff
@@ -43,37 +46,31 @@ func (s *DockerSuite) TestExecInteractiveStdinClose(c *check.C) {
 	if err != nil {
 		c.Fatal(err)
 	}
-
 	contId := strings.TrimSpace(out)
 
-	returnchan := make(chan struct{})
+	cmd := exec.Command(dockerBinary, "exec", "-i", contId, "echo", "-n", "hello")
+	p, err := pty.Start(cmd)
```

Comment:
This is causing a failure on Windows... but seems like something we should be testing on Windows too.

cc @ahmetalpbalkan